### ISSUE TITLE
Typo results in KeyError

### ISSUE THREAD

**Issue by asottile** [PyCQA/flake8#668]
In GitLab by @christian.koehn on Jan 30, 2019, 03:35

Installation in virtual environment:

```
$ pip install flake8
```

`$ flake8 --bug-report`

```json
{
  "dependencies": [
    {
      "dependency": "entrypoints", 
      "version": "0.3"
    }
  ], 
  "platform": {
    "python_implementation": "CPython", 
    "python_version": "2.7.15rc1", 
    "system": "Linux"
  }, 
  "plugins": [
    {
      "is_local": false, 
      "plugin": "mccabe", 
      "version": "0.6.1"
    }, 
    {
      "is_local": false, 
      "plugin": "pycodestyle", 
      "version": "2.5.0"
    }, 
    {
      "is_local": false, 
      "plugin": "pyflakes", 
      "version": "2.1.0"
    }
  ], 
  "version": "3.7.1"
}
```

With the new release we are getting the following exception while executing `flake8 -j 1 src` (I added the `-j 1` for debugging purposes):

```
flake8.checker            MainProcess  44511 DEBUG    Running {'plugin_name': 'pyflakes', 'plugin': <class 'flake8.plugins.pyflakes.FlakesChecker'>, 'name': 'F', 'parameters': OrderedDict([('tree', True), ('file_tokens', True), ('filename', True)]), 'parameter_names': ['tree', 'file_tokens', 'filename']} with {'tree': <_ast.Module object at 0x7f507c0d27d0>}
flake8.checker            MainProcess  44524 CRITICAL Plugin F raised an unexpected exception

Traceback (most recent call last):
  File "/home/ckoehn/.virtualenvs/backend/bin/flake8", line 11, in <module>
    sys.exit(main())
  File "/home/ckoehn/.virtualenvs/backend/local/lib/python2.7/site-packages/flake8/main/cli.py", line 18, in main
    app.run(argv)
  File "/home/ckoehn/.virtualenvs/backend/local/lib/python2.7/site-packages/flake8/main/application.py", line 392, in run
    self._run(argv)
  File "/home/ckoehn/.virtualenvs/backend/local/lib/python2.7/site-packages/flake8/main/application.py", line 380, in _run
    self.run_checks()
  File "/home/ckoehn/.virtualenvs/backend/local/lib/python2.7/site-packages/flake8/main/application.py", line 299, in run_checks
    self.file_checker_manager.run()
  File "/home/ckoehn/.virtualenvs/backend/local/lib/python2.7/site-packages/flake8/checker.py", line 332, in run
    self.run_serial()
  File "/home/ckoehn/.virtualenvs/backend/local/lib/python2.7/site-packages/flake8/checker.py", line 316, in run_serial
    checker.run_checks()
  File "/home/ckoehn/.virtualenvs/backend/local/lib/python2.7/site-packages/flake8/checker.py", line 616, in run_checks
    self.run_ast_checks()
  File "/home/ckoehn/.virtualenvs/backend/local/lib/python2.7/site-packages/flake8/checker.py", line 505, in run_ast_checks
    checker = self.run_check(plugin, tree=ast)
  File "/home/ckoehn/.virtualenvs/backend/local/lib/python2.7/site-packages/flake8/checker.py", line 449, in run_check
    plugin=plugin, excetion=all_exc
  File "/home/ckoehn/.virtualenvs/backend/local/lib/python2.7/site-packages/flake8/exceptions.py", line 86, in __init__
    self.original_exception = kwargs.pop("exception")
KeyError: 'exception'
```

We are actually not sure what the root cause of the plugin exception is, but the parameter [excetion](https://gitlab.com/pycqa/flake8/blob/master/src/flake8/checker.py#L449) is probably a typo :wink:.

**Comment by asottile** [comment]
In GitLab by @christian.koehn on Jan 30, 2019, 03:37

changed the description

**Comment by asottile** [comment]
In GitLab by @carolin.dohmen on Jan 30, 2019, 05:13

mentioned in merge request !291

**Comment by asottile** [comment]
In GitLab by @asottile on Jan 30, 2019, 09:04

This'll be handled by !275

**Comment by asottile** [comment]
In GitLab by @asottile on Jan 30, 2019, 09:04

closed

**Comment by asottile** [comment]
In GitLab by @asottile on Jan 30, 2019, 12:53

This has been released in 3.7.2 -- thanks again for the issue :tada:
